### ISSUE TITLE
Ordered list questions are not accessible via keyboard

### ISSUE THREAD

**Issue by dariiing** [xwiki-labs/cryptpad#1965]
On Forms, keyboard users can't access and answer to this question
![Image](https://github.com/user-attachments/assets/3d12cf48-fae0-4a87-bdcb-b4f157c3fe1d)
